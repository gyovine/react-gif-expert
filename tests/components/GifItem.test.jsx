import { render, screen } from "@testing-library/react";
import { GifItem } from "../../src/components/GifItem";

describe("Prueba snap", () => {
  const title = "chicken";
  const url = "https://www.ricken.com/";
  test("prueba con prop-types", () => {
    const { container } = render(<GifItem title={title} url={url} />);
    expect(container).toMatchSnapshot();
  });
  test("los titles y las img deben coincidir", () => {
    render(<GifItem title={title} url={url} />);
    const { src, alt } = screen.getByRole("img");
    expect(src).toBe(url);
    expect(alt).toBe(alt);
    screen.debug();
  });
});
