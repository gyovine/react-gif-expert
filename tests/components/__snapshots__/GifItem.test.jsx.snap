// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba snap prueba con prop-types 1`] = `
<div>
  <div
    class="Card"
  >
    <img
      alt="chicken"
      src="www.ricken.com"
    />
    <p>
      chicken
    </p>
  </div>
</div>
`;
